tests/model/test_model.py

```diff
@@ -2624,6 +2624,51 @@ def test__handle_reaction_event_for_msg_in_index(
     def update_message_flags_operation(self, request):
         return request.param
 
+    @pytest.mark.parametrize(
+        "event,initial_alerted_words, expected_alert_words",
+        [
+            (
+                {"type": "alert_words", "alert_words": ["word1", "word2", "word3"]},
+                ["word1"],
+                ["word1", "word2", "word3"],
+            ),
+            (
+                {"type": "alert_words", "alert_words": []},
+                ["word1"],
+                [],
+            ),
+            (
+                {"type": "alert_words", "alert_words": ["word1", "word4"]},
+                ["word1"],
+                ["word1", "word4"],
+            ),
+            (
+                {"type": "alert_words", "alert_words": ["word1", "word2"]},
+                ["word1", "word2", "word3"],
+                ["word1", "word2"],
+            ),
+            (
+                {"type": "alert_words", "alert_words": ["word1", "word3"]},
+                ["word1", "word2"],
+                ["word1", "word3"],
+            ),
+        ],
+        ids=[
+            "Add multiple alert words",
+            "Empty alert words",
+            "Add single new alert words",
+            "Delete an alert word",
+            "Add and delete alert words",
+        ],
+    )
+    def test__handle_alert_words_event(
+        self, mocker, model, event, initial_alerted_words, expected_alert_words
+    ):
+        model._handle_alert_words_event(event)
+
+        assert model._alert_words != initial_alerted_words
```

Comment:
Is this always true?

Reply:
Yep, this is always true. But it is more context-dependent. Basically, currently, this test is designed to test the most minimal of changes in the `active_conversation_info` i.e change(`start/stop`) of a single user. The `inital_alerted_words` means just before that change and not at the start of the whole typing event series.
So it is true because of the values that are passed in `mark.parametrize` i.e it is an approach followed in the test.

Reply:
If that is true, then it is good to add a suffix to the test function name to indicate that.

Reply:
I think the name `test__handle_alert_words_event_minimal_change` would do the job.

Reply:
Hmm. Do we get an event if the changes aren't made?

I'd suggest a double underscore from the function name to any 'meaning'.

Reply:
I believe by changes you mean no typing in the writebox(of any client).
That would be a event with `op`=`stop`. So yes.

Reply:
`test__handle_alert_words_event_minimal_change` sounds bit subjective,so i will be changing it to `test__handle_alert_words_event__single_user`. Though it's natural as a single event can't inform about more than 1 user in this case.